### PR TITLE
Pubkey verification & first timestamp

### PR DESCRIPTION
**Pull request by grubach**

### CODE REVIEW

File: src/verification/verification.ts [hunk 1]
```diff
@@ -72,52 +73,79 @@ export class UbirchVerification {
     return transId;
   }
 
+  public getMessenger(): UbirchObservable {
```
Comment (afternoon2) [inline src/verification/verification.ts:76]: Maybe use getter?

```ts
public get messenger(): UbirchObservable {
  return messenger$;
}
```

File: src/verification/verification.ts [hunk 2]
```diff
@@ -246,11 +325,21 @@ export class UbirchVerification {
...
       .map((rawAnchor: IUbirchBlockchainAnchorRAW) => this.createBlockchainAnchor(rawAnchor))
       .filter((probablyAnchor: IUbirchBlockchainAnchor) => probablyAnchor !== undefined);
 
-    return ubirchBlxTxAnchor;
+    return ubirchBlxTxAnchors;
+  }
+
+  protected findFirstAnchorTimestamp(ubirchBlxTxAnchors: IUbirchBlockchainAnchor[]): string | null {
+    const timestamps = ubirchBlxTxAnchors
+      .map(({ timestamp }) => timestamp)
```
Comment (afternoon2) [inline src/verification/verification.ts:337]: maybe `.map(({ timestamp }) => Date.parse(timestamp))` to avoid unnecessary loops?

Comment (grubach) [reply src/verification/verification.ts]: i prefer atomizing layers of abstraction, but ok.

**Review by afternoon2 — APPROVED** [review]

**Review by dieAte — COMMENTED** [review]
ok

**Pull request merged**